### ISSUE TITLE
Evaluate pros/cons, and options for rating projects

### ISSUE THREAD

**Issue by jenlampton** [backdrop-ops/backdropcms.org#1018]
In the meta issue for improving project pages, a 5-star rating was discussed. There may be better options than 5-star that could work for this purpose, and we might want to evaluate some of those.

The first that comes to mind is a thumbs-up / thumbs-down, or maybe even just thumbs-up.

PROS of having ratings:
* could help people trying to decide between modules
* indicates whether a specific project is working or not (for various use-cases?)

CONS of having ratings:
* a low rating can discourage contributors
* required action from people using the projects
* could require lots of maintenance on the site
* may not provide as much value as intended

I'll create a separate issue where we can discuss this one item in more detail.

**Comment by jenlampton** [comment]
The biggest con, for me, would be discouraging contributors who got a low rating on a project. The Backdrop world doesn't work like wordpress where each project costs money and thus has budget for upkeep and improvements.  Here we rely on contributors volunteering their time, and if a project gets a bad score that could reflect poorly on the contributor, or worse, cause them not to want to continue with that project, or even at all.

If we could find a rating system that didn't allow negative feedback, I think that would be preferable.

Something like a thumbs-up only rating system would still allow us to count the thumbs up to rank projects, but would be less likely to negatively affect contribution.

**Comment by yorkshire-pudding** [comment]
> The biggest con, for me, would be discouraging contributors who got a low rating on a project. The Backdrop world doesn't work like wordpress where each project costs money and thus has budget for upkeep and improvements. Here we rely on contributors volunteering their time, and if a project gets a bad score that could reflect poorly on the contributor, or worse, cause them not to want to continue with that project, or even at all.

I totally agree with this point. If I put loads of effort into making a module work and someone negatively rated it because they misunderstood the purpose (happens all the time in retail - people mark down because it didn't do what they thought it would do even though the description is clear), it may dissuade me from putting any more effort in.

We do sometimes get people in the forum who act like they're paying for Backdrop and expect it to be perfect in every way, not recognising that we are all volunteers and may not have time to respond as quickly as they would like.

**Comment by albanycomputers** [comment]
For me, a rating system would not be of any use... 

If I need functionality, and a module seems to offer that functionality I will give it a try... 

If it works for my use case then all is good, should I give it a thumbs up or 5 stars? 

But, if it doesn't work for my use case, would I give it a thumbs down and a 1 star?

Neither would be indicative of the functionality or usability of the module, so I think a rating would be ineffective.

I much prefer the "Date created", "Last updated" and "Usage statistics"...

For me, better descriptions in the readme would be more helpful than a rating system.

Hope that makes sense...

**Comment by jenlampton** [comment]
I'm the same as you @albanycomputers, but there are probably people who don't feel comfortable making that evaluation. I don't know exactly how to make things better for them, but it's certainly worth the discussion.

**Comment by stpaultim** [comment]
> If we could find a rating system that didn't allow negative feedback, I think that would be preferable.

I think the ability to "favorite" modules might fit this criteria and be useful. Just a way for folks to show that they like a specific module and a way to generate a list of "most popular" modules that is different from pure usage statistics. This would not allow for any negative feedback and would be quite simple. Not to mention, it's easy to implement with the flag module and nothing else.

Just a thought.

**Comment by jenlampton** [comment]
I love it. We could then also show a list of each person's favorite projects on their profile :) I think this is a great first step. I'll create a dedicated issue for it...

**Comment by albanycomputers** [comment]
Love it...  that way I can easily add the modules I regularly use to a project without having to (a) remember their names lol and (b) go through each page... going.... "ohhh yeah, I like this one... oh... I forgot about that" :)
